### PR TITLE
Plan Generation for explain analyze describe optimize repair statements 

### PR DESCRIPTION
**Pull request by harshit-gangal**

### CODE REVIEW

File: go/vt/vtgate/plan_executor_test.go [hunk 1]
```diff
@@ -1089,10 +1088,81 @@ func TestPlanExecutorOther(t *testing.T) {
...
+
+	for _, stmt := range stmts {
+		for _, tc := range tcs {
+			sbc1.ExecCount.Set(0)
+			sbc2.ExecCount.Set(0)
+			sbclookup.ExecCount.Set(0)
+
+			_, err := executor.Execute(context.Background(), "TestExecute", NewSafeSession(&vtgatepb.Session{TargetString: tc.targetStr}), stmt, nil)
+			if tc.hasNoKeyspaceErr {
+				assert.Error(t, err, errNoKeyspace)
```
Comment (systay) [inline go/vt/vtgate/plan_executor_test.go:1104]: I think you mean to use `assert.EqualError` here

File: go/vt/vtgate/plan_executor_test.go [hunk 2]
```diff
@@ -1089,10 +1088,81 @@ func TestPlanExecutorOther(t *testing.T) {
...
+			} else if tc.hasDestinationShardErr {
+				assert.Errorf(t, err, "Destination can only be a single shard for statement: %s, got: DestinationExactKeyRange(-)", stmt)
+			} else {
+				assert.NoError(t, err)
+			}
+
+			diff := cmp.Diff(tc.wantCnts, cnts{
```
Comment (systay) [inline go/vt/vtgate/plan_executor_test.go:1111]: `utils.MustMatch`?

File: go/vt/vtgate/planbuilder/other_read.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package planbuilder
```
Comment (systay) [inline go/vt/vtgate/planbuilder/other_read.go:1]: License header

**Comment by harshit-gangal** [comment]
merging as test failure not related to changes.

**Pull request merged**